data/TAGS.txt

```diff
@@ -0,0 +1,146 @@
+# Tags HOWTO
+
+This is a list of used tags in the projects.
+For a tag to be used, it must fulfil all following points:
+- pillars - reflects category/ies and is used to order
+- application verticals
+- technical aspects
+
+The tag-list here is sorted alphabetically, with all tags from one letter comma-separated.
```

Comment:
?

Reply:
That I didn't follow my spec ;) I wanted to have 26 lines, one for each letter, and then comma-separate the tags for each starting letter...